doc/openvas/README.md

```diff
@@ -0,0 +1,4 @@
+# OpenVAS Documentation
+
+To generate thr Table of Content markdown file run `./toc.sh`
```

Comment:
Small typo

Reply:
Something was a broken about this PR, as this file did not even existed anymore in the PR branch. I updated the branch and hopefully it is solved